docs/basics/101-180-FAQ.rst

```diff
@@ -0,0 +1,174 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conceptually and technically, there is no difference between a dataset, a
+subdataset, or a superdataset. The only aspect that makes a dataset a sub- or
+superdataset is whether it is *registered* in another dataset (by means of an entry in the
+``.gitmodules``, automatically performed upon an appropriate ``datalad
+install -d`` command) or contains registered datasets.
+
+How can I convert/import/transform an existing Git or Git-annex repository into a DataLad dataset?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can transform any existing Git or Git-annex repository of yours into a
+DataLad dataset by running::
+
+   $ datalad create -f
+
+inside of it. Afterwards, you may want to tweak settings in ``.gitattributes``
+according to your needs (see sections :ref:`config` and :ref:`config2` for
+additional insights on this).
+
+How can I cite DataLad?
+^^^^^^^^^^^^^^^^^^^^^^^
+
+There is no official paper on DataLad (yet). To cite it, please use the latest
+`zenodo <https://zenodo.org>`_ entry found here:
+`https://zenodo.org/record/3512712 <https://zenodo.org/record/3512712>`_.
+
+What is the difference between DataLad, Git LFS, and Flywheel?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`Flywheel <https://flywheel.io/>`_ is an informatics platform for biomedical
+research and collaboration.
+
+`Git Large File Storage <https://github.com/git-lfs/git-lfs>`_ (Git LFS) is a
+commandline tool that extends Git with the ability to manage large files. In
+that it appears similar to Git-annex.
+
+.. todo::
+
+   TF is flywheel? How can I find out without 2 hours
+
+
+A more elaborate delineation from related solutions can be found in the DataLad
+`developer documentation <http://docs.datalad.org/en/latest/related.html>`_.
+
+DataLad version-controls my large files -- great. But how much is saved in total?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. todo::
+
+   this.
+
+
+How can I copy data out of a DataLad dataset?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Moving or copying data out of a DataLad dataset is always possible and works in
+many cases just like in any regular directory. The only
+caveat exists in the case of annexed data: If file content is managed with
+Git-annex and stored in the :term:`object-tree`, what *appears* to be the
+file in the dataset is merely a symlink (please read section :ref:`symlink`
+to understand this). Moving or copying this symlink will not yield the
+intended result -- instead you will have a broken symlink outside of your
+dataset.
+
+When using the terminal command ``cp``, it is sufficient to use the
+``-L``/``--dereference`` option. This will follow symbolic links, and make
+sure that content gets moved instead of symlinks.
+With tools other than ``cp`` (e.g., graphical file managers), to copy or move
```

Comment:
The absolutely amazing Midnight Commander `mc` can also follow symlinks
![image](https://user-images.githubusercontent.com/136479/68126773-cde0c780-ff14-11e9-8575-3a126efde776.png)